toolkit/docs/building/building.md

```diff
@@ -733,7 +733,8 @@ To reproduce an ISO build, run the same make invocation as before, but set:
 | ARCHIVE_TOOL                  | $(shell if command -v pigz 1>/dev/null 2>&1 ; then echo pigz ; else echo gzip ; fi )                   | Default tool to use in conjunction with `tar` to extract `*.tar.gz` files. Tries to use `pigz` if available, otherwise uses `gzip`
 | INCREMENTAL_TOOLCHAIN         | n                                                                                                      | Only build toolchain RPM packages if they are not already present
 | RUN_CHECK                     | n                                                                                                      | Run the %check sections when compiling packages
-| PACKAGE_BUILD_RETRIES         | 1                                                                                                      | Number of build retries for each package
+| ALLOW_TOOLCHAIN_REBUILDS          |n                                                                                                      | The flag to not report the toolchain rebuild errors. (Used for testing)
```

Comment:
No need to mention that this is now used during testing - it may unnecessarily suggest that its use is narrower than what you can do. The main point of this flag is to not treat toolchain packages rebuilds as errors.
```suggestion
| ALLOW_TOOLCHAIN_REBUILDS          |n                                                                                                      | Do not treat rebuilds of toolchain packages during regular package build phase as errors.
```